samples/VAonAKS/MotherBot/MotherBot/Program.cs

```diff
@@ -26,9 +26,9 @@ public static void Main(string[] args)
                 //// if not standard value of 'ConnectionStrings:AppConfig' will be used
                 .ConfigurationFromAzureAppConfig()
 
-                //// The KeyVault and Certificate Name can be specified as parameters or in ENV 'KeyVaultName' and ENV 'CertificateName'
-                //// For this example it is not needed (no SSL calls to befriended microservices)
-                ////.ConfigureRootCAFromKeyVault()
+                // The KeyVault and Certificate Name can be specified as parameters or in ENV 'KeyVaultName' and ENV 'CertificateName'
+                // For this example it is not needed (no SSL calls to befriended microservices)
+                .ConfigureRootCAFromKeyVault("VmAMaster0711Kv","LocalhostK8s")
```

Comment:
Should be changed to other ENV variables for better reuse

Reply:
I changed those setting to use ENV variables. Please review my change! 💃